src/Host/Broker/Impl/project.json

```diff
@@ -1,20 +1,28 @@
 ﻿{
   "dependencies": {
+    "Libuv": "1.9.0",
     "MicroBuild.Core": "0.2.0",
+    "Microsoft.AspNetCore.Authentication": "1.0.0",
     "Microsoft.AspNetCore.Mvc": "1.0.0",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0",
+    "Microsoft.AspNetCore.Server.Kestrel.Https": "1.0.0",
     "Microsoft.AspNetCore.Server.WebListener": "0.1.0",
     "Microsoft.AspNetCore.WebSockets.Server": "0.1.0",
     "Microsoft.Extensions.Configuration.CommandLine": "1.0.0",
     "Microsoft.Extensions.Configuration.Json": "1.0.0",
     "Microsoft.Extensions.Logging.Console": "1.0.0",
     "Microsoft.Extensions.Logging.Debug": "1.0.0",
+    "Microsoft.Extensions.Logging.TraceSource": "1.0.0",
     "Microsoft.Extensions.Options.ConfigurationExtensions": "1.0.0",
-    "Newtonsoft.Json": "9.0.1"
+    "Microsoft.Tpl.Dataflow": "4.5.24",
+    "Newtonsoft.Json": "9.0.1",
+    "Odachi.AspNetCore.Authentication.Basic": "1.0.0"
```

Comment:
HTTP Basic auth - Kestrel doesn't come with its own. 

---
In reply to: [74828213](https://github.com/Microsoft/RTVS/pull/2173#discussion_r74828213) [](ancestors = 74828213)